README.md

```diff
@@ -30,34 +30,53 @@ guarantee these changes will be backwards-compatible.
 
 ## Usage
 
+### Initial setup
+
 Get the tool via
 
 ```sh
 $ go get -u github.com/golang/dep/cmd/dep
 ```
 
-Typical usage on a new repo might be
+To set up Dep on a project, run the following from your project root directory:
```

Comment:
"To start managing dependencies using `dep`, ..."?

I don't know, but I still think we can come up with something better here. 😁